crates/assists/src/handlers/replace_impl_trait_with_generic.rs

```diff
@@ -0,0 +1,62 @@
+use syntax::ast::{self, edit::AstNodeEdit, make, AstNode, GenericParamsOwner};
+
+use crate::{AssistContext, AssistId, AssistKind, Assists};
+
+// Assist: replace_impl_trait_with_generic
+//
+// Replaces `impl Trait` function argument with the named generic.
```

Comment:
This string should have a before-after example (see other assists for examples). 

We than generate documentation from this example (via `cargo xtask codegen`): https://rust-analyzer.github.io/manual.html#move_bounds_to_where_clause